tests/testthat/test-renderer3-figure-loss-small-data.R

```diff
@@ -0,0 +1,186 @@
+acontext("FigureLossSmall")
+library(data.table)
+library(animint2)
+library(jointseg)
+
+loss.small <- readRDS("loss.small.rds")
```

Comment:
instead of adding another data set to animint2 (which is already almost too big for CRAN), would it be possible to compute these data? if not, a new data set is OK, but please save it as data/something.RData that we can access via `data(something,package="animint2")`

Reply:
It is possible to compute these data instead of storing it. My first attempt at computing the data increases the runtime and memory of the test. Currently it takes 1.3 Gb and 600 seconds on my machine. There may be ways to reduce that by optimizing how loss.small and nb are computed. I will spend some time investigating it:
https://github.com/tdhock/changepoint-data-structure/

Reply:
would be great to use a simple/small version of the data (instead of the full/large data), as long as it captures the essence of the issue

Reply:
not sure how many clickSelects/showSelected values are there, but typically only 2 subsets are necessary for a test

Reply:
I rewrote the test to compute data instead of storing it. Using a small subset of data reduces the runtime. The sample function controls the size of the subset:
sample(1:.N, 5)